BrAddressForm.vue

```diff
@@ -248,8 +268,14 @@ export default {
   }
 };
 // TODO: Move to bedrock-web-forms
+/**
+ * @function isString
+ * @param {String} str
+ * @description checks if something is string.
+ * @returns {Boolean}
```

Comment:
Convention is to have an extra line above @returns see: https://github.com/digitalbazaar/bedrock-account/blob/master/lib/index.js#L94

Reply:
this should be fixed now.